iothub_client/samples/pnp/common/pnp_dps_ll.c

```diff
@@ -2,58 +2,68 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Should this have a deprecated warning as well? Same for other files in this directory.

Reply:
Good question but on this one, no.  The core of the DPS stuff here is mostly the C version of `async await` for the DPS registration to complete which isn't PnP related.  Both the simple thermostat and temp controller use this common code still even after rework.